csrc/scheduler/vectorize_helper.cpp

```diff
@@ -483,6 +483,33 @@ ContiguousInnerDimensionsMapper::computeInfoC2P(
       consumer_ids_to_clear.insert(
           from->getRootDomain().begin(),
           from->getRootDomain().begin() + clear_pos + 1);
+
+      // A special case is that, if the last broadcast dimension resolved in
+      // consumers root domain is mapped for vectorization, then merge order in
+      // the vectorization axes matters.
+      //
+      // T0[i0, i1]
+      // T1[i0, i1, b2] = broadcast(T0)
+      // T2[i0, i1, i3]
+      // T3[i0, i1, i2] = T1 + T2
+      //
+      // If the mapped ids are {i0, i2, i1}, when propagating from T3 to T1, the
+      // previous "inner most resolved broadcast" would eliminate all `i0, i1,
+      // i2` from vectorization, but the reality is that the merged axes has i1
+      // in the inner most and we should still map it for vectorization. see
+      // issue:
```

Comment:
I'm still trying to understand the reasoning here. 

Is vectorizable because the domains are ordered as `{i0, i2, i1}` and the broadcast domain of `i2` only interferes with `i0`?

In this case, is the allocation domains of `T3` are also ordered as `{i0, i2, i1}`?

Reply:
logging some offline conversation: Naoya pointed out a similar issue with the broadcast propagation, tracking that via #1654

There's also a concern on the side effect from this change, i.e. how safe it is regarding the use of vectorization factor in schedulers. tagging @xwang233 @jacobhinkle for comments.